x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-rds-overview.json

```
@@ -0,0 +1,786 @@
+            "panelIndex": "5",
+            "panelRefName": "panel_4",
+            "version": "7.2.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 6,
+              "i": "6",
+              "w": 9,
+              "x": 0,
+              "y": 0
+            },
+            "panelIndex": "6",
+            "panelRefName": "panel_5",
+            "version": "7.2.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 11,
+              "i": "7",
+              "w": 24,
+              "x": 0,
+              "y": 16
+            },
+            "panelIndex": "7",
+            "panelRefName": "panel_6",
+            "version": "7.2.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 11,
+              "i": "8",
+              "w": 24,
+              "x": 24,
+              "y": 16
+            },
+            "panelIndex": "8",
+            "panelRefName": "panel_7",
+            "version": "7.2.0"
+          }
+        ],
+        "timeRestore": false,
+        "title": "[Metricbeat AWS] RDS Overview",
+        "version": 1
+      },
+      "id": "3367c170-921f-11e9-aa19-159bf182e06f",
+      "migrationVersion": {
+        "dashboard": "7.0.0"
+      },
+      "references": [
+        {
+          "id": "17fcda50-921b-11e9-aa19-159bf182e06f",
+          "name": "panel_0",
+          "type": "visualization"
+        },
+        {
+          "id": "45bfd450-921c-11e9-aa19-159bf182e06f",
+          "name": "panel_1",
+          "type": "visualization"
+        },
+        {
+          "id": "8b8a7f80-921c-11e9-aa19-159bf182e06f",
+          "name": "panel_2",
+          "type": "visualization"
+        },
+        {
+          "id": "cc3a1950-921c-11e9-aa19-159bf182e06f",
+          "name": "panel_3",
+          "type": "visualization"
+        },
+        {
+          "id": "00b29040-921d-11e9-aa19-159bf182e06f",
+          "name": "panel_4",
+          "type": "visualization"
+        },
+        {
+          "id": "b5308940-7347-11e9-816b-07687310a99a",
+          "name": "panel_5",
+          "type": "visualization"
+        },
+        {
+          "id": "c1afd130-921e-11e9-aa19-159bf182e06f",
+          "name": "panel_6",
+          "type": "visualization"
+        },
+        {
+          "id": "e06e4cf0-921e-11e9-aa19-159bf182e06f",
+          "name": "panel_7",
+          "type": "visualization"
+        }
+      ],
+      "type": "dashboard",
+      "updated_at": "2019-06-18T23:19:37.398Z",
+      "version": "WzI5NTgsNl0="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "AWS RDS Database Connections",
```

Visualization titles should be suffixed by the module name between brackets
```suggestion
        "title": "Database Connections [AWS RDS]",
```
And once placed in a dashboard, from the dashboard itself, you can edit these titles so they don't have these suffixes, e.g. for this visualization you could change it to `Database Connections`.

---

good to know! Thank you! I will make the changes!